qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -67,6 +69,21 @@ class PhaseGradientUnitary(GateWithRegisters):
 
     Registers:
         phase_grad: A THRU register which the phase gradient is applied to.
+        (optional) ctrl: A THRU register which specifies the control for this gate. Must have
+            `is_controlled` set to `True` to use this register.
+
+    Arguments:
+        bitsize: The number of qubits of the register being acted on
+        exponent: $t$ in the above expression for $\omega_{n, t}$, a multiplicative factor
+            for the phases applied to each state. Defaults to 1.0.
+        is_controlled: `bool` which determines if the unitary is controlled via a `ctrl` register.
+        eps: The precision for the underlying `ZPowGates`.
```

Comment:
Actually, the `eps` passed here is for the state that's being prepared. The `eps` passed to the underlying `ZPowGate` is `eps / self.bitsize` because you apply `self.bitsize` rotations and you want to bound the total error to be less than `eps`.

Reply:
true, updated.